test/util.go

```diff
@@ -96,12 +97,34 @@ func TypeOnCanvas(c fyne.Canvas, chars string) {
 	typeChars([]rune(chars), c.OnTypedRune())
 }
 
+// WaitForThemeToBeApplied waits for the current theme to be applied to the current app.
+func WaitForThemeToBeApplied(t *testing.T) {
+	require.IsType(t, &testApp{}, fyne.CurrentApp())
+	a := fyne.CurrentApp().(*testApp)
+	for a.appliedTheme != a.Settings().Theme() {
+		time.Sleep(1 * time.Millisecond)
+	}
+}
+
 // WidgetRenderer allows test scripts to gain access to the current renderer for a widget.
 // This can be used for verifying correctness of rendered components for a widget in unit tests.
 func WidgetRenderer(wid fyne.Widget) fyne.WidgetRenderer {
 	return cache.Renderer(wid)
 }
 
+// WithTestTheme runs a function with the testTheme temporarily set.
+func WithTestTheme(t *testing.T, f func()) {
```

Comment:
Open question here (and I know I named it this way, but) with the prefix now would `test.WithTheme(...)` be clear enough?

Reply:
I don’t think so. When reading `test.WithTheme(f)` I read that this is the test helper `WithTheme` and it expects at least a `Theme` to use.

Reply:
Ah I see your point. I had imagined that but missed the parameter list sorry :)